utils/sanitiseMetadata.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable camelcase*/
+const { omit, assign } = require("lodash/fp");
+const uuid = require("uuid/v1");
+const EXPIRY_OFFSET_SECONDS = 100;
+
+const filterUnacceptableMeta = omit([
+  "survey_url",
+  "tx_id",
+  "iat",
+  "exp",
+  "eq_id",
+  "form_type",
+  "account_service_url"
+]);
+
+const defaultMetadata = (questionnaireId, tokenIssueTime, surveyUrl) => ({
+  tx_id: uuid(),
+  jti: uuid(),
+  iat: tokenIssueTime,
+  exp: tokenIssueTime + EXPIRY_OFFSET_SECONDS,
+  user_id: "UNKNOWN",
+  case_id: uuid(),
+  ru_ref: "12346789012A",
+  ru_name: "ESSENTIAL ENTERPRISE LTD",
+  trad_as: "ESSENTIAL ENTERPRISE LTD",
+  eq_id: questionnaireId,
+  collection_exercise_sid: uuid(),
+  period_id: "201605",
+  form_type: questionnaireId,
+  survey_url: `${surveyUrl}${questionnaireId}?r${Date.now()}`
```

Comment:
Wonder if `tokenIssueTime` can be used instead of `Date.now()`. Do they produce the same format of output?